src/libraries/System.Text.Encodings.Web/src/System/Text/Encodings/Web/Ssse3Helper.cs

```diff
@@ -13,11 +13,20 @@ internal static class Ssse3Helper
     {
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static Vector128<sbyte> CreateEscapingMask_DefaultJavaScriptEncoderBasicLatin(Vector128<sbyte> sourceValue)
+            => CreateEscapingMask(sourceValue, s_bitMaskLookupBasicLatin, s_bitPosLookup, s_nibbleMaskSByte, s_nullMaskSByte);
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static Vector128<sbyte> CreateEscapingMask(
+            Vector128<sbyte> sourceValue,
+            Vector128<sbyte> bitMaskLookup,
+            Vector128<sbyte> bitPosLookup,
+            Vector128<sbyte> nibbleMaskSByte,
+            Vector128<sbyte> nullMaskSByte)
```

Comment:
I need this arguments in order to be able to hoist these vectors outside the loop (in `TextEncoder`) as the JIT won't do it here.

Codegen for `DefaultJavaScriptEncoderBasicLatin.FindFirstCharacterToEncode` is affected only in minimal way:

```asm
G_M60692_IG11:      ;; bbWeight=4
       C4C17A6F27           vmovdqu  xmm4, xmmword ptr [r15]
       C4C159636710         vpacksswb xmm4, xmm4, xmmword ptr [r15+16]
       C5F828E8             vmovaps  xmm5, xmm0
       C5F828F1             vmovaps  xmm6, xmm1
       C5F828FA             vmovaps  xmm7, xmm2
       C57828C3             vmovaps  xmm8, xmm3
       C5B172D404           vpsrld   xmm9, xmm4, 4
       C531DBCF             vpand    xmm9, xmm9, xmm7
       C5D9DBE7             vpand    xmm4, xmm4, xmm7
       C4E25100E4           vpshufb  xmm4, xmm5, xmm4
       C4C24900E9           vpshufb  xmm5, xmm6, xmm9
       C5D1DBE4             vpand    xmm4, xmm5, xmm4
       C4C15964E8           vpcmpgtb xmm5, xmm4, xmm8
       C5B964E4             vpcmpgtb xmm4, xmm8, xmm4
       C5D1EBE4             vpor     xmm4, xmm5, xmm4
       C5F9D7C4             vpmovmskb eax, xmm4
       85C0                 test     eax, eax
       0F851B010000         jne      G_M60692_IG17
       4983C720             add      r15, 32
       4D3BFD               cmp      r15, r13
       76A8                 jbe      SHORT G_M60692_IG11
```
Those `vmovaps` aren't needed at all*...
In the benchmarks this is just above noise. For larger inputs it's not measurable, for smaller one (just so that vectorization kicks in):
```
|         Method |       TestStringData |      Mean |     Error |    StdDev | Ratio |
|--------------- |--------------------- |----------:|----------:|----------:|------:|
|       PR_42073 | (16, (...)dcsv) [26] |  8.911 ns | 0.0505 ns | 0.0447 ns |  1.00 |
| PR_TextEncoder | (16, (...)dcsv) [26] |  9.085 ns | 0.0178 ns | 0.0158 ns |  1.02 |
|                |                      |           |           |           |       |
|       PR_42073 |   (9, -1, rddnegsne) | 10.245 ns | 0.0445 ns | 0.0372 ns |  1.00 |
| PR_TextEncoder |   (9, -1, rddnegsne) | 10.656 ns | 0.0152 ns | 0.0135 ns |  1.04 |
```
I think this is acceptable (especially if the JIT will be fixed so it doesn't emit these moves...I believe there is already on issue out, but I don't find it rigth now (or is that issue only for scalar-registers?)). 
If not code for `CreateEscapingMask` has to be duplicated.

---

\* if the register allocator would make these actually zero-latency, then there's still the instructions that need to be fetched, decoded, ... and it makes the loop larger.